templates/repo/branch/list.tmpl

```diff
@@ -12,7 +12,12 @@
 			<table class="ui very basic striped fixed table single line">
 				<tbody>
 					<tr>
-						<td>{{.DefaultBranch}}</td>
+						<td>
+							{{if .IsProtected}}
```

Comment:
`.IsProtected` seems to not be defined here. 
You could either search through the branches list for the default branch first or set `ctx.Data['IsProtected']` in the router file.
https://github.com/go-gitea/gitea/blob/a957d4eeac4142e232896c22b45e63de6f21520f/routers/repo/branch.go#L38

Reply:
-> https://github.com/go-gitea/gitea/issues/7576

Reply:
ok you mean the protection symbol never shows up eaven it have to on defaultbranch
-> ill work on this